public/app/features/alerting/unified/Insights.tsx

```diff
@@ -0,0 +1,141 @@
+  type: 'prometheus',
+  uid: 'grafanacloud-usage',
+};
+
+const grafanaCloudPromDs = {
+  type: 'prometheus',
+  uid: 'grafanacloud-prom',
+};
+
+const THIS_WEEK_TIME_RANGE = new SceneTimeRange({ from: 'now-1w', to: 'now' });
+const LAST_WEEK_TIME_RANGE = new SceneTimeRange({ from: 'now-2w', to: 'now-1w' });
+
+function getGrafanaScenes() {
+  return new EmbeddedScene({
+    body: new SceneFlexLayout({
+      wrap: 'wrap',
+      children: [
+        getMostFiredInstancesScene(THIS_WEEK_TIME_RANGE, ashDs, 'Top 10 firing instances this week'),
+
+        getFiringAlertsRateScene(THIS_WEEK_TIME_RANGE, ashDs, 'Alerts firing per minute'),
+
+        getFiringAlertsScene(THIS_WEEK_TIME_RANGE, ashDs, 'Firing alerts this week'),
+
+        getFiringAlertsScene(LAST_WEEK_TIME_RANGE, ashDs, 'Firing alerts last week'),
+      ],
+    }),
+  });
+}
+
+function getCloudScenes() {
+  return new EmbeddedScene({
+    body: new SceneFlexLayout({
+      wrap: 'wrap',
+      children: [
+        getAlertsByStateScene(THIS_WEEK_TIME_RANGE, cloudUsageDs, 'Alerts by State'),
+        getNotificationsScene(THIS_WEEK_TIME_RANGE, cloudUsageDs, 'Notifications'),
+
+        getSilencesScene(LAST_WEEK_TIME_RANGE, cloudUsageDs, 'Silences'),
+        getInvalidConfigScene(THIS_WEEK_TIME_RANGE, cloudUsageDs, 'Invalid configuration'),
+      ],
+    }),
+  });
+}
+
+function getMimirManagedRulesScenes() {
+  return new EmbeddedScene({
+    body: new SceneFlexLayout({
+      wrap: 'wrap',
+      children: [
+        getMostFiredCloudInstances(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, 'Top 5 firing instance this week'),
+        getFiringCloudAlertsScene(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, 'Firing'),
+        getPendingCloudAlertsScene(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, 'Pending'),
+
+        getInstancesByStateScene(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, 'Count of alert instances by state'),
+        getInstancesPercentageByStateScene(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, '% of Alert Instances by State'),
+
+        getEvalSuccessVsFailuresScene(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, 'Evaluation Success vs Failures'),
+        getMissedIterationsScene(THIS_WEEK_TIME_RANGE, grafanaCloudPromDs, 'Iterations Missed'),
+      ],
+    }),
+  });
+}
+
+function getMimirManagedRulesPerGroupScenes() {
+  return new EmbeddedScene({
+    body: new SceneFlexLayout({
+      wrap: 'wrap',
+      children: [
+        //Rule group evaluation
+        //Rule group interval
+        //rule group evaluation duration
+        //rules per group
+      ],
+    }),
+  });
+}
+
+export default function Insights() {
+  const styles = useStyles2(getStyles);
+  const grafanaScenes = getGrafanaScenes();
+  const cloudScenes = getCloudScenes();
+  const mimirRulesScenes = getMimirManagedRulesScenes();
+  const mimirRulesPerGroupScenes = getMimirManagedRulesPerGroupScenes();
+
+  return (
+    <>
+      <div className={styles.container}>
+        <h4>Grafana</h4>
```

Comment:
nit: I think we generally discourage the usage of `H(N)` in favor of the `<Text variant='h4>Hello, world!</Text>` if we can't guarantee the document will also have a H1 through H3 preceding it.